src/test/package/nuget/NuGetTest/Directory.Build.targets

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project TreatAsLocalProperty="OutDir;IntDir" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+    <!-- This file is automatically imported by MSBuild for each project in under this folder, 
+         see https://docs.microsoft.com/en-us/visualstudio/msbuild/customize-your-build -->
+
+    <Import Condition="Exists($([MSBuild]::GetPathOfFileAbove('Directory.Build.targets', '$(MSBuildThisFileDirectory)../')))" Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.targets', '$(MSBuildThisFileDirectory)../'))" />
```

Comment:
>../ [](start = 117, length = 3)

I think this is a mistake (same for second one).  It's mentioned on the docs.microsoft.com page above, where it's also a mistake.  In my experiments of chaining directory.build.* file imports, the GetPathOfFileAbove does what you want and expect - the ".." is redundant, and maybe incorrect - have you tested?

Reply:
It works this way, but it might be redundant.